programs/zstdcli.c

```diff
@@ -748,6 +749,7 @@ int main(int const argCount, const char* argv[])
                 if (!strcmp(argument, "--priority=rt")) { setRealTimePrio = 1; continue; }
                 if (!strcmp(argument, "--output-dir-flat")) {nextArgumentIsOutDirName=1; lastCommand=1; continue; }
                 if (!strcmp(argument, "--show-default-cparams")) { showDefaultCParams = 1; continue; }
+                if (!strcmp(argument, "--no-content-size")) { noContentSize = 1; continue; }
```

Comment:
Consider adding both `--content-size` and `--no-content-size`, 
with `--content-size` being the default.
If both are present, the latest wins.
This can happen when accumulating options on the command line through macros and scripts.